project/themes/default/index.php

```diff
@@ -155,7 +155,7 @@
 
 <h3>
  
-<span id="taskview" class="mtt-menu-button"><span class="btnstr"><?php _e('tasks');?></span> (<span id="total">0</span>) <span class="arrdown"></span></span>
+<!--<span id="taskview" class="mtt-menu-button"><span class="btnstr"><?php _e('tasks');?></span> (<span id="total">0</span>) <span class="arrdown"></span></span> -->
```

Comment:
1.did you test it on your local machine ? 
2.did you sync up with designer leader about the front-end design ?